src/server/status/server_status.js

```diff
@@ -55,12 +55,13 @@ export default class ServerStatus {
   }
 
   overall() {
-    const state = _(this._created)
-    .map(function (status) {
-      return states.get(status.state);
-    })
-    .sortBy('severity')
-    .pop();
+    const state = Object
+      // take all created status objects
+      .values(this._created)
+      // get the state descriptor for each status
+      .map(status => states.get(status.state))
+      // reduce to the state with the highest severity, defaulting to green
+      .reduce((a, b) => a.severity > b.severity ? a : b, states.get('green'));
```

Comment:
That's a great point, if we have statuses and they are all disabled... does that make the overall status disabled or green? I'm tempted to say green, because disabled status should pretty much mean "ignore me"

Reply:
There are places where we check that `status === 'green'` to make sure things are good, I don't think we should be reporting anything else if all statuses are disabled.

Reply:
Thanks for checking, sounds reasonable to me.